docs/readmes/contributing/contribute_onboarding.md

```diff
@@ -97,15 +97,40 @@ Install Magma locally and get everything running.
 **Howto**
 
 1. Follow the [prerequisites guide](https://magma.github.io/magma/docs/next/basics/prerequisites) and install all development tools, up to but not including the "Build/Deploy Tooling" section
-2. Run all Orc8r tests
+2. (Optional) If you opt to use IntelliJ IDEA as your local IDE, follow the instructions in the "Set up IntelliJ" section below before you proceed
```

Comment:
You can provide the direct link to this. Double check it (using the create_docusarus_website.sh script), but should be something like
```suggestion
2. (Optional) If you opt to use IntelliJ IDEA as your local IDE, follow the instructions in the [Set up IntelliJ](#set-up-intellij) section below before you proceed
```

Reply:
Doubled checked & verified that it works!